packages/vscode-vue/src/common.ts

```diff
@@ -143,7 +143,7 @@ async function doActivate(context: vscode.ExtensionContext, createLc: CreateLang
 			'Reload Window'
 		);
 		if (reload === undefined) return; // cancel
-		vscode.commands.executeCommand('workbench.action.reloadWindow');
+		vscode.commands.executeCommand('workbench.action.restartExtensionHost');
```

Comment:
this is faster than reloading whole window as it reloads extension host only (instead of reloading other core vscode things as well)

for example in some cases it can save you from losing terminals state